packages/cli-lib/github.js

```diff
@@ -19,17 +20,26 @@ const GITHUB_AUTH_HEADERS = {
  * @param {String} repoName - name of the github repository
  * @returns {Buffer|Null} Zip data buffer
  */
-async function fetchJsonFromRepository(repoName, filePath) {
+async function fetchJsonFromRepository(
+  repoName = 'HubSpot/hubspot-project-components',
+  filePath,
+  repoPath = false
+) {
   try {
-    const URI = `https://raw.githubusercontent.com/HubSpot/${repoName}/${filePath}`;
+    const URI = `https://raw.githubusercontent.com/${repoName}/${filePath}`;
     logger.debug(`Fetching ${URI}...`);
 
-    return request.get(URI, {
+    return await request.get(URI, {
       json: true,
       headers: { ...DEFAULT_USER_AGENT_HEADERS, ...GITHUB_AUTH_HEADERS },
     });
   } catch (err) {
     logger.error('An error occured fetching JSON file.');
+    if (repoPath && err.statusCode === 404) {
```

Comment:
One last small nit. The console is noisy when we hit this error. Maybe we shouldn't log `logger.error('An error occurred fetching JSON file.');` when this check hits? So we only log one or the other. Here's an example of the console:

<img width="844" alt="image" src="https://github.com/HubSpot/hubspot-cli/assets/6654014/37504be0-0d9c-43fc-894e-4835a4ea5d91">

Reply:
Fixed in https://github.com/HubSpot/hubspot-cli/pull/837/commits/3d549ea4cb140ba9d558e364c834c2dd7aa11a9c